validphys2/src/validphys/plotutils.py

```diff
@@ -25,13 +26,59 @@
 
 log = logging.getLogger(__name__)
 
+def subplots(figsize=None,nrows=1, ncols =1,sharex=False, sharey=False):
```

Comment:
Consider adding `**kwargs` and pass them down directly to `fig.subplots()`.

In particular, you could treat this way all the arguments, since the defaults are already present in [`Figure.subplots()`](https://matplotlib.org/stable/api/figure_api.html#matplotlib.figure.Figure.subplots) and you do not manipulate them at all.

```suggestion
def subplots(figsize=None, **kwargs):
```
and
```py
ax = fig.subplots(**kwargs)
```

Reply:
I think this is a good idea. It makes the change even more transparent.